features/step_definitions/ability_steps.rb

```diff
@@ -16,7 +16,7 @@
   click_on "Create Ability"
 end
 
-When("I visit character show page") do
+When("I visit character Trol show page") do
   visit user_path(@registered_user)
   click_on "Trol"
 end
```

Comment:
"Visit some page" step should include only `visit page_path` helper if possible.

Reply:
This comment isn't implemented.